### DISCUSSION TITLE
Install php_pgsql

### DISCUSSION THREAD

**Discussion by gilvaju** [asdf-community/asdf-php#75] [Q&A] [answered]
I need use php_pgsql.

Is possible install on asdf installation?

**Comment by smorimoto** [comment]
Doesn't this work?

```bash
PHP_CONFIGURE_OPTIONS="--with-pgsql" asdf install php <VERSION>
```

or

```bash
PHP_CONFIGURE_OPTIONS="--with-pgsql=<PATH TO PSQL>" asdf install php <VERSION>
```

**Comment by gilvaju** [comment]
The installation dont finished.

I use: `PHP_CONFIGURE_OPTIONS="--with-pgsql" asdf install php 7.4.12`

![Captura de Tela_Área de Seleção_20201028155612](https://user-images.githubusercontent.com/677039/97495878-53902b00-194f-11eb-9753-022eefecb2f4.png)

**Comment by smorimoto** [comment]
Please tell me the environmental information such as OS, its version, etc...

**Comment by smorimoto** [comment]
If you are using macOS, have you done all these steps?

```bash
brew install autoconf automake bison freetype gettext icu4c krb5 libedit libiconv libjpeg libpng libxml2 libzip pkg-config re2c zlib

export PKG_CONFIG_PATH="$(brew --prefix icu4c)/lib/pkgconfig:$(brew --prefix krb5)/lib/pkgconfig:$(brew --prefix libedit)/lib/pkgconfig:$(brew --prefix libxml2)/lib/pkgconfig:$(brew --prefix openssl)/lib/pkgconfig"

export PATH="$(brew --prefix bison)/bin:$PATH"

PHP_CONFIGURE_OPTIONS="--with-pgsql" asdf install php 7.4.12
```

**Comment by smorimoto** [comment, marked as answer]
Probably because openssl is not found, if the following log appears during configure:

```console
checking for OpenSSL support... no
```

For example, I use Void Linux, so I had to specify it like this:

```bash
PHP_CONFIGURE_OPTIONS="--with-pgsql --with-openssl=/usr/include/openssl" asdf install php 7.4.12
```

**Comment by gilvaju** [comment]
Hi, i use arch linux

**Comment by gilvaju** [comment]
> Probably because openssl is not found, if the following log appears during configure:
> 
> ```
> checking for OpenSSL support... no
> ```
> 
> For example, I use Void Linux, so I had to specify it like this:
> 
> ```shell
> PHP_CONFIGURE_OPTIONS="--with-pgsql --with-openssl=/usr/include/openssl" asdf install php 7.4.12
> ```

I will try --with-openssl

![Captura de Tela_Área de Seleção_20201029065441](https://user-images.githubusercontent.com/677039/97565121-d0adb580-19cc-11eb-930f-466d2957a049.png)

**Comment by gilvaju** [comment]
`PHP_CONFIGURE_OPTIONS="--with-pgsql --with-openssl=/usr/include/openssl" asdf install php 7.4.12`

It's work. Thank you very much.

**Comment by smorimoto** [comment]
I'm glad it works. Your question seems to have been resolved, so I will close this issue.
